model/FeatureFlag/UniqueNumericFeatureFlag.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2024 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\model\FeatureFlag;
+
+interface UniqueNumericFeatureFlag
+{
+    public const FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER = 'FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER';
```

Comment:
**chore:** There is no need to add FF to constants. They are basically unique works already, if we are following the standard like `FEATURE_FLAG_*`. Also, they can fluctuate among several extensions, which makes trick to have a central place to create these constants. 

Moreover, you already document it on README.md and they should also be documented here https://github.com/oat-sa/nextgen-stack/blob/main/docs/feature-flags.md, please :) 

Also, I am not sure it is useful to create interface just to hold constants and not defining any behavior.